ansible_collections/arista/avd/roles/eos_designs/docs/tables/l3-edge.md

```diff
@@ -422,7 +426,7 @@
           # The peer device interface. Used for description and documentation
           peer_interface: <str>
 
-          # The peer device IP. Used for description and documentation
+          # The peer device IP/Mask. Used as default route gateway is `set_default_route` is true and `ip` is an IP address.
```

Comment:
```suggestion
          # The peer device's IP/Mask. Used as default route gateway if `set_default_route` is true and `ip` is an IP address.
```

Reply:
Should this be ip/mask or just ip?

Reply:
given that it is IP/Mask for the `ip` key, it is weird to have it as IP only on the other side. Or maybe should I use mask in another place?

Reply:
[703afd1](https://github.com/aristanetworks/ansible-avd/pull/3440/commits/703afd1054dca9d5d00eb04a41b1059f4f810265)

Reply:
will remove mask